src/data/items/resourceChemistry.js

```diff
@@ -77,9 +85,9 @@ const POTIONS = {
 		name: "Energy Drink",
 		sellPrice: 79,
 		icon: require("@/assets/art/chemistry/energy_drink.png"),
-		description: "Whenever you get energy from engineering, also get four times that amount of money.",
+		description: "Whenever you get energy from engineering, also get ten times that amount of money.",

```

Comment:
dat stealth buff tho

Reply:
I asked Nabski about this before doing it, I really should make a comprehensive changelog somewhere in this PR though